spec/lib/traject/config/item_info_spec.rb

```diff
@@ -86,24 +87,9 @@
     end
 
     it 'skips invalid buildings' do
-      buildings = []
-      results.map do |result|
-        buildings << result[field]
-      end
-
-      expect(buildings.flatten).not_to include(
-        'APPLIEDPHY', # Applied Physics Department
-        'CPM', # 1391080 GREEN - Current Periodicals & Microtext
-        'GRN-REF', # 2442876, GREEN - Reference - Obsolete
-        'ILB', # 1111, Inter-Library Borrowing
-        'SPEC-DESK', # GREEN (Humanities & Social Sciences)
-        'SUL',
-        'PHYSICS', # Physics Library
-        'MEYER', # INDEX-168 Meyer
-        'BIOLOGY', # closed
-        'CHEMCHMENG', # closed
-        'MATH-CS' # closed
-      )
+      invalid_buildings = results.select { |item| item[field].nil? }
+      ids_for_invalid_buildings = invalid_buildings.map { |n| n['id'].first }
+      expect(ids_for_invalid_buildings).to eq %w[1111 2222 115472 460947 919006 1391080 1732616 2442876 2797607 3142611 3277173 6493823 7117119 10421123]
```

Comment:
I'm not sure this is a case that can happen any more, but I'm confused about what this rewritten test is testing. Previously, it was clear that those random buildings weren't present... and now, I'm not sure?

Reply:
I was concerned that it was looking for the non-presence of some codes. However, all the other values in that array were decoded, human readable labels.  This seems like it probably wasn't a good test, so instead I'm testing for all of the record ids in the results  where the building_facet is nil.